platform/o.n.bootstrap/nbproject/project.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 javac.compilerargs=-Xlint:all -Xlint:-serial -Xlint:-processing
-javac.source=1.8
+javac.release=11
```

Comment:
note to self: this bumps bootstrap to 11 to have access to the `StackWalker` API.

this should be fine since `Main` is compiled separately, so the warning dialogs etc will still show up when launched on older JDKs:
https://github.com/apache/netbeans/blob/33cdacf4e3d5591b41e3d2971f7bfab800ddf464/platform/o.n.bootstrap/build.xml#L32-L40

verified with launch attempt on JDK 8:
![image](https://github.com/user-attachments/assets/976a6345-ae21-4318-b286-5e9a1aca3d47)